Raven.setExtraContext is not exposed in typescript definition

The `setExtraContext` function is not present in the Typescript definition.

```Typescript
Raven.setExtraContext({ phoneNumber: Customer.PhoneNumber })
```
`src/component/App.tsx(167,13): error TS2339: Property 'setExtraContext' does not exist on type 'RavenStatic'.`

https://github.com/getsentry/raven-js/blob/4a8591b7ae5a22dc100322f058f8dfde57f802c2/typescript/raven.d.ts

--- a/src/raven.js
+++ b/src/raven.js
@@ -381,6 +381,7 @@ Raven.prototype = {
         if (this._breadcrumbs.length > this._globalOptions.maxBreadcrumbs) {
             this._breadcrumbs.shift();
         }
+        return this;
     },
 
     addPlugin: function(plugin /*arg1, arg2, ... argN*/) {

--- a/typescript/raven-tests.ts
+++ b/typescript/raven-tests.ts
@@ -46,10 +46,24 @@ Raven.setUserContext({
     id: '123'
 });
 
+Raven.setExtraContext({foo: 'bar'});
+Raven.setTagsContext({env: 'prod'});
+Raven.clearContext();
+var obj:Object = Raven.getContext();
+var err:Error = Raven.lastException();
+
 Raven.captureMessage('Broken!');
 Raven.captureMessage('Broken!', {tags: { key: "value" }});
+Raven.captureBreadcrumb({});
+
+Raven.setRelease('abc123');
+Raven.setEnvironment('production');
 
 Raven.setDataCallback(function (data) {});
 Raven.setDataCallback(function (data, original) {});
 Raven.setShouldSendCallback(function (data) {});
 Raven.setShouldSendCallback(function (data, original) {});
+
+Raven.showReportDialog({
+    eventId: 'abcdef123456'
+});

--- a/typescript/raven.d.ts
+++ b/typescript/raven.d.ts
@@ -154,6 +154,9 @@ interface RavenStatic {
      */
     captureMessage(msg: string, options?: RavenOptions): RavenStatic;
 
+    /** Log a breadcrumb */
+    captureBreadcrumb(crumb: Object): RavenStatic;
+
     /**
      * Clear the user context, removing the user data that would be sent to Sentry.
      */
@@ -171,8 +174,29 @@ interface RavenStatic {
         email?: string;
     }): RavenStatic;
 
+    /** Merge extra attributes to be sent along with the payload. */
+    setExtraContext(context: Object): RavenStatic;
+
+    /** Merge tags to be sent along with the payload. */
+    setTagsContext(tags: Object): RavenStatic;
+
+    /** Clear all of the context. */
+    clearContext(): RavenStatic;
+
+    /** Get a copy of the current context. This cannot be mutated.*/
+    getContext(): Object;
+
     /** Override the default HTTP data transport handler. */
     setTransport(transportFunction: (options: RavenTransportOptions) => void): RavenStatic;
+
+    /** Set environment of application */
+    setEnvironment(environment: string): RavenStatic;
+
+    /** Set release version of application */
+    setRelease(release: string): RavenStatic;
+
+    /** Get the latest raw exception that was captured by Raven.*/
+    lastException(): Error;
 
     /** An event id is a globally unique id for the event that was just sent. This event id can be used to find the exact event from within Sentry. */
     lastEventId(): string;
@@ -185,6 +209,9 @@ interface RavenStatic {
 
     /** Specify a callback function that allows you to apply your own filters to determine if the message should be sent to Sentry. */
     setShouldSendCallback(data: any, orig?: any): RavenStatic;
+
+    /** Show Sentry user feedback dialog */
+    showReportDialog(options: Object);
 }
 
 interface RavenTransportOptions {

